### PR TITLE
wb: bands intensity fix

### LINKED ISSUE #169: NSF Exercise 3 - western blot bands are incorrect
I haven't gone through all of the bands in detail, but bands are appearing where there shouldn't be any bands. For example, the WT EGFR sample with the FLAG or His antibodies should be blank. @annagav, would you like to go through the coding Excel spreadsheet together?

### PR DESCRIPTION
**Pull request by annagav**
Fix #169

### CODE REVIEW

File: html_app/assignments/scb_ex/assignment.js [hunk 1]
```diff
@@ -4906,7 +4906,7 @@ var __scb_sample_3 = {
                                 {
                                     name: 'Mouse anti-pEGFR',
                                     weight: 140,
-                                    intensity: 8,
```
Comment (carsongee) [inline html_app/assignments/scb_ex/assignment.js:4909]: I get the switch above (not that I really understand it), but why the intensity change too?

**Comment by carsongee** [comment]
Code wise, looks fine, just a question for my understanding.  I tried to duplicate the screen shot from the issue, and the bands look different, so I guess it works ;) LOL

File: html_app/assignments/scb_ex/assignment.js (hunk 1 above)
Comment (annagav) [inline html_app/assignments/scb_ex/assignment.js:4909]: it had wrong intensity value. Not sure if that answers the question.

Comment (carsongee) [inline html_app/assignments/scb_ex/assignment.js:4909]: Oh, I missed that in the issue.  I see it now.

**Comment by carsongee** [comment]
:kimono:

**Comment by annagav** [comment]
Is there anything else that needs to fixed, @carsongee ?

**Comment by carsongee** [comment]
Nope, missed the merge button there, sorry.

**Pull request merged**